src/main/bicep/mainTemplate.bicep

```diff
@@ -86,9 +86,26 @@ param newOrExistingVnetForSingleServer string = 'new'
 @description('To mitigate ARM-TTK error: Control Named vnetForSingleServer must output the resourceGroup property when hideExisting is false')
 param vnetRGNameForSingleServer string = resourceGroup().name
 
+@description('Boolean value indicating, if user wants to enable database connection.')
+param enableDB bool = false
+@allowed([
+  'db2'
+])
+@description('One of the supported database types')
+param databaseType string = 'db2'
+@description('JNDI Name for JDBC Datasource')
+param jdbcDataSourceJNDIName string = 'jdbc/contoso'
+@description('JDBC Connection String')
+param dsConnectionURL string = 'jdbc:db2://contoso.db2.database:50000/sample'
+@description('User id of Database')
+param dbUser string = 'contosoDbUser'
+@secure()
+@description('Password for Database')
+param dbPassword string = newGuid()
+
 param guidValue string = newGuid()
 
-var const_arguments = format(' {0} {1}', wasUsername, wasPassword)
+var const_arguments = format(' {0} {1} {2} {3} {4} {5} {6} {7}', wasUsername, wasPassword, enableDB, databaseType, base64(jdbcDataSourceJNDIName), base64(dsConnectionURL), base64(dbUser), base64(dbPassword))
```

Comment:
See my forthcoming email "Subject: Career leadership technical development idea: apply knowledge across the workstreams: for example: env vars vs. positional parameters"